source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -464,3 +464,10 @@ def event_UIA_notification(
 		for line in displayString.splitlines():
 			if line and not line.isspace():  # Don't say "blank" during autoread
 				speech.speakText(line)
+
+#: WinTerminalUIA is kept for backwards compatibility.
+WinTerminalUIA = (
+	NotificationsBasedWinTerminalUIA
+	if _shouldUseWindowsTerminalNotifications()
```

Comment:
what happens if this config changes during run time?

Reply:
Now with the use of `getattr`, this value updates at runtime.